bitnami/ruby/3.0/debian-11/Dockerfile

```diff
@@ -3,6 +3,14 @@ ENV OS_ARCH="amd64" \
     OS_FLAVOUR="debian-11" \
     OS_NAME="linux"
 
+LABEL org.opencontainers.image.authors="https://bitnami.com/contact" \
+      org.opencontainers.image.description="Application packaged by Bitnami" \
+      org.opencontainers.image.ref.name="3.0.4-debian-11-r27" \
+      org.opencontainers.image.source="https://github.com/bitnami/containers/tree/main/bitnami/ruby" \
+      org.opencontainers.image.title="ruby" \
+      org.opencontainers.image.vendor="VMware, Inc." \
+      org.opencontainers.image.version="3.0.4"
+
```

Comment:
@carrodher There is something weird `3.0.4` vs `3.1.2` in the README file.

Reply:
This asset supports 3 branches, in the README there is an entry per branch, in this case `3.1.x`, `3.0.x` and `2.7.x`.
This PR is updating the `3.0.x` branch, for that branch, I think everything matches: PR title, changes in the _Dockerfile_, correct `ruby/3.0` folder, etc